### PR TITLE
polish texts on campus forms and remove page indications

### LINKED ISSUE #249: Polish campus form - step 1
Screen Shot 2019-04-26 at 10 36 16

Instead of `Tel. number:` => `Telephone number:`

Instead of `Email:` => `E-mail address:`

Instead of `Maximum number of Students per year in campus` => `Annual student capacity`

### LINKED ISSUE #251: Remove "Page x of x" from Campus and Qualification forms
Screenshot 2019-04-23 at 21 01 53

It is redundant. Let's just remove it from the campus and qualification form.

### LINKED ISSUE #252: Polish campus form - step 2
Screen Shot 2019-04-26 at 10 47 11

Instead of `Physical Address` => `Physical address:`
Instead of `Postal Address` => `Postal address:`

### LINKED ISSUE #253: Polish campus form - step 3
Screen Shot 2019-04-26 at 10 50 13

Instead of `Event name` => `Event name:`
Instead of `Starting date:` => `Start date:`
Instead of `Ending date:` => `End date:`

### PR DESCRIPTION
**Pull request by boney-bun**
fix #251
fix #252
fix #253
fix #249

### CODE REVIEW

**Comment by boney-bun** [comment]
PIA:
ford3-polishtexts

**Comment by pierrealixt** [comment]
Campus form - step 3: add a `:` after `Link to event`.

File: django_project/ford3/forms/campus.py [hunk 1]
```diff
@@ -21,21 +21,21 @@ class CampusDetailForm(CampusForm):
     #     required=False)
 
     telephone = forms.CharField(
-        label='Tel. Number:',
+        label='Telephone Number:',
```
Comment (pierrealixt) [inline django_project/ford3/forms/campus.py]: Telephone number. No capital letter at `number`.

**Review by pierrealixt — COMMENTED** [review]
`Telephone number:` instead of `Telephone Number:`

**Review by pierrealixt — APPROVED** [review]
LGTM

**Pull request merged**